sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -1276,9 +1312,13 @@ Accept wildcard characters: False
 ```
 
 ### -UserVoiceForFeedbackEnabled
+Enables or disables the User Voice Feedback button.
+
 PARAMVALUE: $true | $false
 
-When set to $true, the "Feedback" link will be shown at the bottom of all modern SharePoint Online pages. The "Feedback" link will allow the end user to fill out a feedback form inside SharePoint Online which will then create an entry in the public SharePoint UserVoice topic. When set to $false, feedback link will not be shown anymore. It may take up to an hour for a change of this property to be reflected consistently throughout your tenant.
+When set to $true, the "Feedback" link will be shown at the bottom of all modern SharePoint Online pages. The "Feedback" link will allow the end user to fill out a feedback form inside SharePoint Online which will then create an entry in the public SharePoint UserVoice topic. 
+
+When set to $false, feedback link will not be shown anymore. It may take up to an hour for a change of this property to be reflected consistently throughout your tenant.
```

Comment:
```suggestion
When set to $false, The Feedback link will not be shown anymore. It may take up to an hour for a change of this property to be reflected consistently throughout your tenant.
```